content-src/components/Base/Base.jsx

```diff
@@ -87,7 +88,10 @@ export class _Base extends React.PureComponent {
     if (prefs["asrouter.devtoolsEnabled"]) {
       if (window.location.hash.startsWith("#asrouter") ||
           window.location.hash.startsWith("#devtools")) {
-        return (<ASRouterAdmin />);
+        return (<div>
+          <ASRouterAdmin />
+          <ASRouterUISurface dispatch={this.props.dispatch} />
```

Comment:
Ideally it would be great not to render this in two places, although I think for now this is probably your best option